{{cookiecutter.repo_name}}/Makefile

```diff
@@ -36,14 +48,30 @@ requirements:
 	pip install -qr requirements.txt --exists-action w
```

Comment:
@cpennington I vote we return to using the `requirements` target for local development, and `prod-requirements` for production. You've broken my muscle memory!